src/privateai_client/components/pai_uris.py

```diff
@@ -1,13 +1,26 @@
+import logging
+
+
 class PAIURIs:
-    def __init__(self, scheme, host, port=None):
-        self.valid_schemes = ["http", "https"]
-        scheme = scheme.split("://")[0]
-        if scheme not in self.valid_schemes:
+    def __init__(self, url=None, scheme=None, host=None, port=None, **kwargs):
+        if url:
+            self._pai_uri = url
+        elif scheme and host:
+            logging.warning(
+                'Deprecation warning. scheme, host and port are no longer supported. Please use url when initializing the client. Eg. PAIClient(url="http://localhost:8080")'
```

Comment:
Hey, @koryf and @mikeyoungPAI actually preferred scheme, host and port. I think we can allow users to specify both?

Reply:
ah ok! I've removed the deprecation warning so both can be used :)